code/datums/uplink/grenades.dm

```diff
@@ -33,10 +33,10 @@
 	item_cost = 2
 	path = /obj/item/storage/box/emps
 	desc = "A box of five grenades that cause a risky EMP explosion, capable of toggling headsets off, permanently destroying IPC units and draining a stationbound completely."
-	
+
 /datum/uplink_item/item/grenades/frag
 	name = "5xFrag Grenades"
-	item_cost = 6
```

Comment:
Imho the current price is fine as is.
Especially when the grenades are being buffed.